#!/usr/bin/python3
def raise_exception():
    i = "2"
    j = 3
    result = i / j
    return result
